process/start_project.md

```diff
@@ -104,4 +125,26 @@ The Academy Software Foundation staff is happy to engage and advise during this
 
 ### What happens next?
 
-Generally, the TAC will schedule project proposals at the following [TAC Meeting](/meetings). Guidelines for the presentation to the TAC are detailed in the [project lifecycle](/process/lifecycle#tac-presentation).
+Generally, the TAC will schedule project proposals at the following [TAC Meeting](/meetings), provided there is at least one week between the submitted project proposal and the scheduled TAC meeting.
+
+Proposed projects will have a 30-minute presentation timeslot at an upcoming TAC meeting as the schedule allows. The project proposal must include a presentation conforming to the structure below to ensure uniform and complete submissions.
+
+*   Overview of the project and its purpose
+    *  If a project is new, it’s often helpful to share the vision for the project, the anticipated structure, and benefits.
+    *  If the project is an existing open-source project proposed to join ASWF, it is often helpful to schedule a project walkthrough for the community to understand better the project, including the architecture, structure, and how to get started using the project. This often helps review progress more efficiently.
```

Comment:
We haven't really seen such a presentation yet (that I can remember), project proposals have been more treated as a black box. I would personally be interested in something like that (kind of a mini code review), but 30 minutes goes by fast...

Reply:
I think part of it is most of the projects proposed are already quite known; slowly, we are seeing newer projects that aren't familiar to the TAC being proposed, and yes, having a better understanding of how the project works would be helpful, not only for evaluating but also gaining contributors.